Tools/AP_Bootloader/board_types.txt

```diff
@@ -44,6 +44,8 @@ TARGET_HW_ARK_CAN_RTK_GPS              82
 TARGET_HW_FF_RTK_CAN_GPS               85
 TARGET_HW_ATL_MANTIS_EDU               97
 TARGET_HW_ARK_FMU_V6X                  57
+TARGET_HW_THE_PEACH_K1                212
```

Comment:
please can you use the 1024+ range - that is where we are putting new flight controllers

Reply:
it is a common ID with PX4, so 212 is fine